docs/install/_linux.rst

```diff
@@ -136,6 +109,35 @@ To change the system service owner, you need to change the configuration of the
 .. note:: Replace the ``$USER`` in commands above with the name of the user you wish to be the new Kolibri system service owner.
 
 
+Optimized Performance with ``kolibri-server`` package
+-----------------------------------------------------
+
+We finalized the release of an optimized version of Kolibri for Linux systems called ``kolibri-server``. This version is recommended for:
+
+* Clear server-client setup for implementations with many client devices
```

Comment:
> Clear server-client setup

Not sure what this means

Reply:
I believe it was supposed to differentiate from single device installation, that might not have clients to serve.

Reply:
specifically I don't know what the word "Clear" means in this context

Reply:
I think you can remove the text about what it's recommended for. That will leave the information from the first paragraph and the one preceding the list: Any system with Debian/Ubuntu, gives 2-3x the performance on a Raspberry Pi. I used "2-3x" in the example to be a bit conservative. But I believe @jredrejo has indicated that it might be more like 3-4x from benchmarks.

Reply:
Removed!